### DISCUSSION TITLE
TOP Ranking Miner f0127595 ProveCommitSector fee is 0 FIL. Why is the fee free?

### DISCUSSION THREAD

**Discussion by ktg49292** [filecoin-project/lotus#6234] [Storage Provider] [answered]
I'm looking at the rankings of other miners,
Looking at the top miners, the ProveCommitSector fee is displayed as 0 FIL. 

I would like to know how this is possible. 
I pay about 0.24 FIL for all ProveCommitSectors. 

Isn't it unfair?

The link address is as follows:
https://filfox.info/en/address/f0429122

<img width="1217" alt="스크린샷 2021-05-12 오전 9 37 24" src="https://user-images.githubusercontent.com/82360290/117901863-36c5ae80-b307-11eb-90d4-cbb30ab193a4.png">
<img width="969" alt="스크린샷 2021-05-12 오전 9 39 54" src="https://user-images.githubusercontent.com/82360290/117901873-3af1cc00-b307-11eb-9b52-e69a2e637c93.png">

**Comment by hyunmoon** [comment, marked as answer]
Filfox simply shows you how much has been transferred into the miner actor.
But because they're paying directly from the miner's available balance, no transfer was made hence 0FIL is displayed.
